dev_guide/routes.adoc

```diff
@@ -270,6 +270,37 @@ xref:../architecture/networking/routes.adoc#path-based-routes[path-based
 routing] are available in the
 xref:../architecture/networking/routes.adoc#architecture-core-concepts-routes[Architecture section].
 
+[[dev-guide-routes-allowing-endpoints-to-control-cookies]]
+== Allowing Route Endpoints to Control Cookie Names
+
+{product-title} provides sticky sessions, which enables stateful application
+traffic by ensuring all traffic hits the same endpoint. However, if the endpoint
+pod terminates, whether through restart, scaling, or a change in configuration,
+this statefulness can disappear.
+
+{product-title} can use cookies to configure session persistence. The router selects an endpoint to handle any user requests, and creates a cookie for the session. The cookie is passed back in the response to the request and the user sends the cookie back with the next request in the session. The cookie tells the router which endpoint is handling the session, ensuring that client requests use the cookie so that they are routed to the same pod.
```

Comment:
Would this be better?
s/{product-title} can use cookies/{product-title} uses cookies